app/graphql/types/digital_object_interface.rb

```diff
@@ -57,14 +57,14 @@ def resources
       end
     end
 
-    def self.title_for(object)
+    def self.title_for(object, indexable = false)
```

Comment:
It really feels like this method should be moved to DigitalObject.

Reply:
I think the only reason it hasn't is that we were talking about making it an actual core field

Reply:
This might be a good time to move it because when that change happens the implementation of this method can be updated and graphql or the indexing code wouldn't know the difference.